CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.17.3 (2021-08-11)
+
+- Fixed the lookup styles in Firefox and Safari. [#211](https://github.com/blackbaud/skyux-lookup/pull/211)
```

Comment:
Is the fix that we are referring to just the `?` that we add to the country field and lookup `.ts` files below? We are linking to this PR, so just making sure that's the change we are referring to since it doesn't seem specific to Firefox or Safari.

Reply:
@blackbaud-johnly the fix is in the update to the theme package, but the build demonstrated a regression that occurred in the tests — theme service is an optional dependency on country field and lookup but there is an error if theme service is omitted — and the `?` is the small fix.